components/BugsnagTask.brs

```diff
@@ -243,7 +243,11 @@ function createDevicePayload()
 		device["deviceId"] = deviceInfo.GetChannelClientId()
 	end if
 
-	device["firmwareVersion"] = deviceInfo.GetOSVersion()
+	if FindMemberFunction(deviceInfo, "GetOSVersion") <> invalid
+		device["firmwareVersion"] = deviceInfo.GetOSVersion()
+	else 
+		device["firmwareVersion"] = deviceInfo.GetVersion()
```

Comment:
The docs don't look convincing that this will be the same thing as OS version. Have you been able to check it on a device?

If it's not the OS version, maybe we can just report "unknown" or "pre-9.2" when `GetOSversion()` is not available?

Reply:
You are right, these is not the same. I could either do some string manipulation to end up with a similar structure, although it would be missing a revision number.

Now I have realized, `GetOSVersion` returns an object while `GetVersion` returns a string. So I will fix this so that object is set either way for firmwareVersion, but will put revision to `n/a` or something similar. What do you think?

Reply:
Sounds good :+1:

Reply:
Refactored in 525284ff ™️ 

Now, either way, an object is returned. Only difference is that for firmware lower than 9.2, revision number is `n/a`.